communication/marketing-team/role-handbooks/wip-roles.md

```diff
@@ -0,0 +1,12 @@
+# Work In Progress Roles
+
+## Purpose  
+Capture roles that we would like to have but don't have them filled for Reasons.
+
+
+## Roles
+1. Designer
+someone to help with youtube thumbnails, logos, social clips/gifs  
+help us manage digital assets  
+
+2. TODO / Add one here
```

Comment:
Do we want to think about this for a role: https://github.com/kubernetes/community/issues/4726

Reply:
good idea! but let's make it official on the main README :)

once the PR i have in flight is merged with the templates and handbook link references from the readme, i'll add you

Reply:
/lgtm

Reply:
/lgtm

Reply:
LGTM except the spelling of liaision. Missed that :)

Reply:
Yay!!!